eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/Arbitrary.hs

```diff
@@ -652,9 +652,8 @@ instance Era era => Arbitrary (ShelleyPpupPredFailure era) where
   arbitrary = genericArbitraryU
   shrink = genericShrink
 
-instance Era era => Arbitrary (ShelleyPoolPredFailure era) where
+instance (Arbitrary (TxCert era), Era era) => Arbitrary (ShelleyPoolPredFailure era) where
   arbitrary = genericArbitraryU
-  shrink = genericShrink
```

Comment:
This resulted in a bizarre overlapping generic instances error. So, I chose to remove shrinking instead of enabling IncoherentInstances